nelua/cdefs.lua

```diff
@@ -189,9 +192,90 @@ cdefs.target_info_code = [[
 #if defined(__ANDROID__)
   is_android = true;
 #endif
+#if defined(APPLE_IOS) || defined(IOS) || defined(__IPHONEOS__) || defined(TARGET_OS_SIMULATOR) || defined(__IOS__) || defined(__APPLE_IOS__) || defined(IOS_OBJ_C) || defined(TARGET_OS_IOS)
+  is_ios = true;
+#endif
 #if defined(__MACH__)
   is_mach = true;
 #endif
+#if defined(__HPUX__)
+  is_hpux = true;
+#endif
+#if defined(__SGI__)
+  is_irix = true;
+#endif
+#if defined(__TIZEN_H__) || defined(TIZEN_DEPRECATION) || defined(__TIZEN__)
```

Comment:
Seems like `__TIZEN_H__` will come from a header, and no header is included when preprocessing this file, so this looks like seems, I think `__TIZEN_H__` and `TIZEN_DEPRECATION` could be removed, leaving only `__TIZEN__`.

Reply:
This done via commit in the patch now!